content/messaging/index.mdx

```diff
@@ -116,24 +121,24 @@ If you're not sure, which implementation to choose, we suggest reading about par
 The LiveChat Messaging API adopts the following versioning convention:
 
 - **stable**
-  
+
   The latest stable version we encourage developers to use. We don't introduce any new functionalities or breaking changes - only bug fixes are expected.
 
 - **developer preview (dev preview)**
-  
+
   An unstable version that provides a preview of the upcoming features. **It's not open to public use.** However, if you want to test some features, contact us at developers@livechat.com or <a href="#open-chat" onClick={openChatWindow}>ask on the chat</a>, and we'll give you access.
 
 - **legacy**
-  
+
   The old version we still support but highly encourage you to switch to the latest stable version.
 
 **Currently:**
 
-|                 | Agent Chat API | Customer Chat API | Configuration API |
```

Comment:
I removed Config API from this table to be consistent. I think we should add a similar table to Reports API.